src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media_Animation/SetTarget_After_Start.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml_Media_Animation
+{
+	[TestFixture]
+	public partial class SetTarget_After_Start : SampleControlUITestBase
+	{
+		[Test]
+		[AutoRetry]
+		public async Task When_Target_Is_Set_After_Start()
+		{
+			Run("UITests.Shared.Windows_UI_Xaml_Media_Animation.SetTargetProperty");
+			var playBtn = _app.Marked("playButton");
+
+			var animatedRect = _app.Query("AnimatedRect");
+			var animatedRectRect = animatedRect.Single().Rect.ToRectangle();
+
+			var container = _app.Query("Container");
+			var containerRect = container.Single().Rect.ToRectangle();
+
+			// 5 is tolerance.
+			Assert.LessOrEqual(Math.Abs(animatedRectRect.X - containerRect.X), 5);
```

Comment:
This would give a better error message, should it ever fail:
```suggestion
const int Tolerance = 5;
Assert.AreEqual(containerRect.X, animatedRectRect.X, Tolerance);
```